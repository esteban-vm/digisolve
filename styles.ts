import { css } from '@emotion/react'

export const globalStyles = css`
  @import url('/css/fluid.css');
  @import url('/css/normalize.min.css');

  @font-face {
    font-family: 'Work Sans';
    src: local('Work Sans'), url('/fonts/WorkSans-VariableFont_wght.ttf');
  }

  :root {
    --color-primary: #ed07b7;
    --color-dark-primary: #bd0592;
    --color-secondary: #6920b0;
    --color-tertiary: #acdaff;

    --color-white: #fff;
    --color-grey: #555;
    --color-black: #000;

    --color-light: #f4f4f4;
    --color-lighter: #fcfcfc;

    --color-overlay1: #0000004d;
    --color-overlay2: #000000b3;
    --color-overlay3: #00000080;
    --color-overlay4: #000000cc;

    --color-shadow: #efefefef;
  }

  html,
  body {
    background-color: var(--color-white);
    color: var(--color-black);
    text-rendering: optimizeLegibility;
    font-size: 16px;
    font-family: 'Work Sans', Arial, sans-serif;
    font-weight: 300;
  }

  .row {
    max-width: 1140px;
    margin: 0 auto;
  }

  .clear_fix {
    zoom: 1;

    ::after {
      content: '';
      display: table;
      clear: both;
      visibility: hidden;
    }
  }
`

export * from '@emotion/react'
export { default as styled } from '@emotion/styled'
