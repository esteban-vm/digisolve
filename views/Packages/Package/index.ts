export { default, type PackageProps } from './Package'
