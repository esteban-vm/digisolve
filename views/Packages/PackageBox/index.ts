export { default, type PackageProps } from './PackageBox'
