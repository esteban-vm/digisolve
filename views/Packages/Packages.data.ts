import type { PackageProps } from './Package'

const description = 'Ratione similique aut incidunt repellendus excepturi ut commodi laboriosam.'

const packages: PackageProps[] = [
  {
    id: 'd7e645e0-e825-11ed-b24d-893619994521',
    title: 'starter',
    price: 699,
    duration: 3,
    description,
    features: [
      'SEO, SMO and PPC',
      '2 Blogs per month',
      'Facebook & Twitter Marketing',
      'Monthly Performance',
      'Traffic Report',
    ],
  },
  {
    id: 'df095c40-e825-11ed-b24d-893619994521',
    title: 'corporate',
    price: 2999,
    duration: 6,
    description,
    features: [
      'SEO, SMO and PPC',
      '4 Blogs per month',
      'All Social Media Channels',
      'Monthly Performance',
      'Traffic Report',
    ],
  },
  {
    id: 'e21563c0-e825-11ed-b24d-893619994521',
    title: 'enterprice',
    price: 4999,
    duration: 6,
    description,
    features: [
      'Personalized Marketing',
      '8 Blogs per month',
      'All Social Media Channels',
      'Monthly Performance',
      'Traffic Report',
    ],
  },
]

export default packages
