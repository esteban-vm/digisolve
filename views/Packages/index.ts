export { default as Packages } from './Packages'
