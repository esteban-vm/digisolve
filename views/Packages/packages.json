[
  {
    "title": "starter",
    "price": 699,
    "duration": 3,
    "features": [
      "SEO, SMO and PPC",
      "2 Blogs per month",
      "Facebook & Twitter Marketing",
      "Monthly Performance",
      "Traffic Report"
    ]
  },
  {
    "title": "corporate",
    "price": 2999,
    "duration": 6,
    "features": [
      "SEO, SMO and PPC",
      "4 Blogs per month",
      "All Social Media Channels",
      "Monthly Performance",
      "Traffic Report"
    ]
  },
  {
    "title": "enterprice",
    "price": 4999,
    "duration": 6,
    "features": [
      "Personalized Marketing",
      "8 Blogs per month",
      "All Social Media Channels",
      "Monthly Performance",
      "Traffic Report"
    ]
  }
]
