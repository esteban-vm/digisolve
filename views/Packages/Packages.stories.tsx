import type { Meta } from '@/stories'
import Packages from './Packages'

export default {
  component: Packages,
  title: 'Views/Packages',
  parameters: {
    backgrounds: {
      disable: true,
    },
  },
} as Meta<typeof Packages>

export const View = () => <Packages />
