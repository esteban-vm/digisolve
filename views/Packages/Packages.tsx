import type { Component } from '@/types'
import { Grid } from '@/components'
import { uuid } from '@/helpers'
import { styled, mediaQuery } from '@/styles'
import packages from './packages.json'
import Package from './Package'

const PackagesComponent: Component = (props) => {
  return (
    <section id='packages' aria-labelledby='packages_title' {...props}>
      <Grid.Row>
        <h2 id='packages_title'>Check out our most popular packages</h2>
      </Grid.Row>
      <Grid.Row>
        {packages.map((_package) => (
          <Grid.Col key={uuid()} isOneThird>
            <Package {..._package} />
          </Grid.Col>
        ))}
      </Grid.Row>
    </section>
  )
}

const Packages = styled(PackagesComponent)`
  text-align: center;
  background-color: var(--color-light);

  h2 {
    color: var(--color-black);
    font-size: 250%;
    margin-bottom: 20px;
    margin-top: 150px;
    text-transform: capitalize;

    ${mediaQuery('lg')} {
      margin-top: 90px;
    }

    ${mediaQuery('sm')} {
      margin-top: 60px;
    }
  }
`

export default Packages
