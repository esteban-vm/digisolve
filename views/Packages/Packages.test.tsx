import { render, cleanup, screen, create } from '@/tests'
import { mediaQuery } from '@/styles'
import packages from './Packages.data'
import Packages from './Packages'

describe('ðŸ§ª PACKAGES:', () => {
  describe('display tests:', () => {
    /** Section title */
    const name = /^check out our most popular packages$/i

    beforeEach(() => {
      render(<Packages />)
    })

    afterEach(cleanup)

    it('should be accessible', () => {
      const section = screen.getByRole('region', { name })
      expect(section).toBeInTheDocument()
    })

    describe('should be displayed:', () => {
      it('the title', () => {
        const title = screen.getByRole('heading', { name, level: 2 })
        expect(title).toBeInTheDocument()
        expect(title).toBeVisible()
      })

      it('the packages', () => {
        const packageElements = screen.getAllByRole('article')
        expect(packageElements).toHaveLength(packages.length)

        for (const pack of packageElements) {
          expect(pack).toBeInTheDocument()
          expect(pack).toBeVisible()
        }
      })
    })
  })

  describe('style tests:', () => {
    let tree: ReturnType<typeof create>

    beforeEach(() => {
      tree = create(<Packages />)
    })

    it('should render with correct styles', () => {
      expect(tree).toMatchSnapshot()
    })

    describe('should have correct styles on:', () => {
      const property = 'margin-top'
      const target = 'h2'

      it('small screen devices', () => {
        expect(tree).toHaveStyleRule(property, '60px', { target, media: mediaQuery('sm') })
      })

      it('large screen devices', () => {
        expect(tree).toHaveStyleRule(property, '90px', { target, media: mediaQuery('lg') })
      })
    })
  })
})
