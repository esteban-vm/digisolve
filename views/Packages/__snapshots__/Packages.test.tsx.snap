// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ðŸ§ª PACKAGES: style tests: should render with correct styles 1`] = `
.emotion-0 {
  text-align: center;
  background-color: var(--color-light);
}

.emotion-0 h2 {
  color: var(--color-black);
  font-size: 250%;
  margin-bottom: 20px;
  margin-top: 150px;
  text-transform: capitalize;
}

@media only screen and (max-width: 1200px) {
  .emotion-0 h2 {
    margin-top: 90px;
  }
}

@media only screen and (max-width: 768px) {
  .emotion-0 h2 {
    margin-top: 60px;
  }
}

.emotion-1 {
  background-color: var(--color-white);
  width: 90%;
  margin: 0 auto 30px;
  box-shadow: 0 3px 2px var(--color-shadow);
  text-align: left;
}

@media only screen and (max-width: 1024px) {
  .emotion-1 {
    width: 100%;
  }
}

@media only screen and (max-width: 480px) {
  .emotion-1 {
    width: 80%;
  }
}

.emotion-1 h4 {
  font-size: 150%;
  color: var(--color-grey);
  font-weight: 400;
  margin-bottom: 50px;
  text-transform: capitalize;
}

@media only screen and (max-width: 1024px) {
  .emotion-1 h4 {
    margin-bottom: 25px;
  }
}

.emotion-1 p:first-of-type {
  color: var(--color-primary);
  font-size: 300%;
  font-weight: 300;
  margin-bottom: 10px;
}

.emotion-1 p:first-of-type span {
  font-size: 30%;
  font-weight: 500;
}

.emotion-1 p:last-of-type {
  color: var(--color-grey);
  font-size: 80%;
  margin-bottom: 10px;
}

.emotion-1 ul li {
  color: var(--color-grey);
  padding: 5px 0;
}

.emotion-1 .section {
  padding: 15px;
  border-bottom: 1px solid var(--color-shadow);
}

.emotion-1 .section:first-of-type {
  background-color: var(--color-lighter);
}

.emotion-1 .section:last-of-type {
  border: 0;
  text-align: center;
}

.emotion-1 .icon {
  width: 1.2rem;
  height: 1.2rem;
  display: inline-block;
  margin-right: 10px;
  color: var(--color-primary);
}

.emotion-1 .btn {
  margin: 15px 0;
}

.emotion-2 {
  display: inline-block;
}

.emotion-2 input[type='submit'],
.emotion-2 a {
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  text-transform: capitalize;
  padding: 10px 30px;
  border-radius: 200px;
  font-weight: 500;
  outline: none;
  -webkit-transition: background-color 500ms;
  transition: background-color 500ms;
  border: 2px solid var(--color-primary);
  background-color: var(--color-primary);
  color: var(--color-white);
}

.emotion-2 input[type='submit']:hover,
.emotion-2 a:hover,
.emotion-2 input[type='submit']:active,
.emotion-2 a:active {
  background-color: var(--color-dark-primary);
}

<section
  aria-labelledby="packages_title"
  className="emotion-0"
  id="packages"
>
  <div
    className="row "
    role="presentation"
  >
    <h2
      id="packages_title"
    >
      Check out our most popular packages
    </h2>
  </div>
  <div
    className="row "
    role="presentation"
  >
    <div
      className="col span_1_of_3 "
      role="presentation"
    >
      <article
        aria-labelledby="package_title"
        className="emotion-1"
      >
        <div
          className="section"
        >
          <h4
            id="package_title"
          >
            starter
          </h4>
          <p>
            $699
             
            <span>
              3
               months
            </span>
          </p>
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Modi, sit nemo natus ad vero molestias?
          </p>
        </div>
        <div
          className="section"
        >
          <ul>
            <li>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-check icon"
                data-icon="check"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={{}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M470.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L192 338.7 425.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                  fill="currentColor"
                  style={{}}
                />
              </svg>
              SEO, SMO and PPC
            </li>
            <li>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-check icon"
                data-icon="check"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={{}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M470.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L192 338.7 425.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                  fill="currentColor"
                  style={{}}
                />
              </svg>
              2 Blogs per month
            </li>
            <li>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-check icon"
                data-icon="check"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={{}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M470.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L192 338.7 425.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                  fill="currentColor"
                  style={{}}
                />
              </svg>
              Facebook & Twitter Marketing
            </li>
            <li>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-check icon"
                data-icon="check"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={{}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M470.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L192 338.7 425.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                  fill="currentColor"
                  style={{}}
                />
              </svg>
              Monthly Performance
            </li>
            <li>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-check icon"
                data-icon="check"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={{}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M470.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L192 338.7 425.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                  fill="currentColor"
                  style={{}}
                />
              </svg>
              Traffic Report
            </li>
          </ul>
        </div>
        <div
          className="section"
        >
          <span
            className="btn emotion-2"
          >
            <a
              href="#"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
              role="button"
            >
              Sign up now
            </a>
          </span>
        </div>
      </article>
    </div>
    <div
      className="col span_1_of_3 "
      role="presentation"
    >
      <article
        aria-labelledby="package_title"
        className="emotion-1"
      >
        <div
          className="section"
        >
          <h4
            id="package_title"
          >
            corporate
          </h4>
          <p>
            $2,999
             
            <span>
              6
               months
            </span>
          </p>
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Modi, sit nemo natus ad vero molestias?
          </p>
        </div>
        <div
          className="section"
        >
          <ul>
            <li>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-check icon"
                data-icon="check"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={{}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M470.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L192 338.7 425.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                  fill="currentColor"
                  style={{}}
                />
              </svg>
              SEO, SMO and PPC
            </li>
            <li>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-check icon"
                data-icon="check"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={{}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M470.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L192 338.7 425.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                  fill="currentColor"
                  style={{}}
                />
              </svg>
              4 Blogs per month
            </li>
            <li>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-check icon"
                data-icon="check"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={{}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M470.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L192 338.7 425.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                  fill="currentColor"
                  style={{}}
                />
              </svg>
              All Social Media Channels
            </li>
            <li>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-check icon"
                data-icon="check"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={{}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M470.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L192 338.7 425.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                  fill="currentColor"
                  style={{}}
                />
              </svg>
              Monthly Performance
            </li>
            <li>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-check icon"
                data-icon="check"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={{}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M470.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L192 338.7 425.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                  fill="currentColor"
                  style={{}}
                />
              </svg>
              Traffic Report
            </li>
          </ul>
        </div>
        <div
          className="section"
        >
          <span
            className="btn emotion-2"
          >
            <a
              href="#"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
              role="button"
            >
              Sign up now
            </a>
          </span>
        </div>
      </article>
    </div>
    <div
      className="col span_1_of_3 "
      role="presentation"
    >
      <article
        aria-labelledby="package_title"
        className="emotion-1"
      >
        <div
          className="section"
        >
          <h4
            id="package_title"
          >
            enterprice
          </h4>
          <p>
            $4,999
             
            <span>
              6
               months
            </span>
          </p>
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Modi, sit nemo natus ad vero molestias?
          </p>
        </div>
        <div
          className="section"
        >
          <ul>
            <li>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-check icon"
                data-icon="check"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={{}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M470.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L192 338.7 425.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                  fill="currentColor"
                  style={{}}
                />
              </svg>
              Personalized Marketing
            </li>
            <li>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-check icon"
                data-icon="check"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={{}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M470.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L192 338.7 425.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                  fill="currentColor"
                  style={{}}
                />
              </svg>
              8 Blogs per month
            </li>
            <li>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-check icon"
                data-icon="check"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={{}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M470.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L192 338.7 425.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                  fill="currentColor"
                  style={{}}
                />
              </svg>
              All Social Media Channels
            </li>
            <li>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-check icon"
                data-icon="check"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={{}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M470.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L192 338.7 425.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                  fill="currentColor"
                  style={{}}
                />
              </svg>
              Monthly Performance
            </li>
            <li>
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-check icon"
                data-icon="check"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={{}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M470.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L192 338.7 425.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
                  fill="currentColor"
                  style={{}}
                />
              </svg>
              Traffic Report
            </li>
          </ul>
        </div>
        <div
          className="section"
        >
          <span
            className="btn emotion-2"
          >
            <a
              href="#"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onTouchStart={[Function]}
              role="button"
            >
              Sign up now
            </a>
          </span>
        </div>
      </article>
    </div>
  </div>
</section>
`;
